src/utils.js

```diff
@@ -112,3 +114,20 @@ export const requestPromise = (options, iteration = 0) => {
         });
     });
 };
+
+/**
+ * Checks that given parameter is of given type and throws ApifyError.
+ * If errorMessage is not provided then error message is created from name and type of param.
+ *
+ * @param type         string - "String", "Number", ... (see ee: https://github.com/gkz/type-check)
+ * @param value        string - user entered value of that parameter
+ * @param name         string - parameter name (crawlerId for options.crawlerId)
+ * @param errorMessage string - optional error message
+ */
+export const checkParamOrThrow = (type, value, name, errorMessage) => {
```

Comment:
@jancurn I think it's good to have parameters `value` and `name` next to each other as they "look" so similar.

`checkParamOrThrow('String', baseUrl, 'baseUrl');` is better than `checkParamOrThrow(baseUrl, String, 'baseUrl');`

In the second case it would be easy to modify one and forget about the other one and get wrong error messages.

It's a minor thing, just my 2 cents...